docs/_snippets/applies_to-key.md

```diff
@@ -0,0 +1,36 @@
+`applies_to` accepts the following keys in this structure:
+
+* `stack`: Applies to the [Elastic Stack](https://www.elastic.co/docs/get-started/the-stack) including any Elastic Stack components.
```

Comment:
In [best-practices.md](https://github.com/elastic/docs-builder/pull/1614/files#diff-f49f84500f068e2077460fcf7eedbd5c606679a93a73da6eaef7c44ad0c263a7) you talk about keys needing to be in the same order for consistency. Should this file be the source of truth on what that order is?